zdm/parameters.py

```diff
@@ -22,19 +22,12 @@ class AnalysisParams(data_class.myDataClass):
 # Beam parameters
 @dataclass
 class BeamParams(data_class.myDataClass):
-    thresh: int = field(
-        default=0,
-        metadata={'help': '??'})
-    method: int = field(
+    Bmethod: int = field(
         default=2,
         metadata={'help': 'Method for calculation. See beams.py:simplify_beam() for options'})
```

Comment:
This field is auto-magically slurped into the Table for the paper. 
  so would be nice to give explicity info when possible.

Reply:
What is "the table"? I can certainly improve this (lots of the code needs better documentation) but this is tangential to this pull request, all I am doing here is renaming a thing